notebooks/202-vision-superresolution/202-vision-superresolution-video.ipynb

```diff
@@ -222,9 +214,9 @@
     "original_image_key, bicubic_image_key = compiled_model.inputs\n",
     "output_key = compiled_model.output(0)\n",
     "\n",
-    "# Get the expected input and target shape. `.dims[2:]` returns the height\n",
-    "# and width. OpenCV's resize function expects the shape as (width, height),\n",
-    "# so we reverse the shape with `[::-1]` and convert it to a tuple\n",
+    "# Get the expected input and target shape. The `.dims[2:]` returns the height\n",
```

Comment:
```suggestion
    "# Get the expected input and target shape. The `.dims[2:]` function returns the height\n",
```

Reply:
.dims is not a function. I recommend not adding words like functions when you are not certain. These words have a very specific meaning.